test/binCases/watch/info-verbosity-off/info-verbosity-off.test.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const { runWatch } = require("../../../testUtils");
+
+test("info-verbosity-off", () => {
+	return runWatch(__dirname, [
```

Comment:
async-await?

Reply:
We still support node 6 and we don't use Babel. I would have used already if I could

Reply:
Jest comes with babel support out of the box (that's how code coverage works), so you can just add `babel-preset-env` and use it in tests if you want, even if you don't want to transpile your source code. Up to you, though 🙂

Reply:
Preset env is doable, but will it throw on node v6 and below?

Reply:
nah, just use `node: 'current'` as `target`

Reply:
I wouldn't mind to use babel for your test suite honestly